src/components/SearchForm/SearchForm.js

```diff
@@ -17,7 +18,7 @@ function SearchForm() {
       <input
         onChange={handleChange}
         type='text'
-        placeholder='Enter topic'
+        placeholder={placeholder}
         value={'' || search}

```

Comment:
` value={'' || search}`


you need to check vise versa to be logically correct


```js

 value={search  || '' }

```